Mediapipe.Net/Calculators/ICalculator.cs

```diff
@@ -7,14 +7,33 @@
 
 namespace Mediapipe.Net.Calculators
 {
+    /// <summary>
+    /// This interface allows for a simple use of any MediaPipe calculator.
+    /// </summary>
+    /// <typeparam name="T">The landmarks return type of the calculator graph.</typeparam>
     public interface ICalculator<T> : IDisposable
     {
+        /// <summary>
+        /// Starts the calculator.
+        /// </summary>
         public void Run();
 
+        /// <summary>
+        /// Sends an <see cref="ImageFrame"/> to MediaPipe to process.
+        /// </summary>
+        /// <remarks>If the input <see cref="ImageFrame"/> doesn't get disposed, MediaPipe will crash.</remarks>
+        /// <param name="frame">The frame that MediaPipe should process.</param>
+        /// <returns>An <see cref="ImageFrame"/> with the contents of the source <see cref="ImageFrame"/> and the landmarks drawn.</returns>
         public ImageFrame Send(ImageFrame frame);
 
+        /// <summary>
+        /// This event triggers every time MediaPipe is able to detect landmarks on a given <see cref="ImageFrame"/>.
+        /// </summary>
         public event EventHandler<T> OnResult;
 
+        /// <summary>
+        /// The current processed frame number.
+        /// </summary>
         public long CurrentFrame { get; }
     }
```

Comment:
When I think about it, do we even need this interface? The only class that implements it is `Calculator`.

Reply:
yes we do because you can initialize either the CPU or GPU version of the same graph by storing it as an `ICalculator<T>`

Reply:
Yet again, we have `Calculator` for that.

Reply:
we do not

Reply:
... Wait, you removed it? I'm pretty sure I made that class

Reply:
you probably didn't commit. anyway, an interface here is better than an abstract class because both implementations are different

Reply:
I mean yeah but there's still a lot of repetition there.
Maybe I confused `Calculator` with `CpuCalculator`, since that was already an abstraction of all the code that was being repeated in every other calculator...
Anyways, I'll try to make one. It'll probably come in handy.